cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -678,6 +678,22 @@
     </description>
   </property>
 
+  <property>
+    <name>data.local.storage.user.table.max.open.files</name>
+    <value>100</value>
+    <description>
+      Maximum number of open files per user table.
+    </description>
+  </property>
+
+  <property>
+    <name>data.local.storage.system.table.max.open.files</name>
+    <value>500</value>
+    <description>
+      Maximum number of open files per system table.
```

Comment:
Maximum number of open file descriptors per system table to be used for local storage.

Reply:
While re-writing this description, the thing that is not clear to me is if this config is per system table and across all system tables?

Reply:
I think the "per system table" makes the distinction clear.